proxy/proxy-tests/src/test/scala/io/cloudstate/proxy/stress/CrdtStressSpec.scala

```diff
@@ -266,7 +266,7 @@ object CrdtStressSpec {
         |akka.jvm-exit-on-fatal-error = false
       """.stripMargin).withFallback(ConfigFactory.defaultReference()).resolve()
     implicit val system = ActorSystem("cloudstate-proxy", config)
-    implicit val materializer = ActorMaterializer()
+    implicit val materializer = Materializer
```

Comment:
There seems to be able 3 different ways of obtaining a materializer used in this PR, why and when should those 3 be used?

Reply:
I'll fix. Sorry for inconsistency, I hate that!

Reply:
@sean-walsh This uses `Materializer` and not `Materializer(system)`, why?

Reply:
A typo and this was unused, removing.